src/sql/workbench/browser/designer/designer.ts

```diff
@@ -957,6 +953,27 @@ export class Designer extends Disposable implements IThemable {
 					table.registerPlugin(deleteRowColumn);
 					columns.push(deleteRowColumn.definition);
 				}
+				if (tableProperties.canInsertRows || tableProperties.canMoveRows) {
+					const moreActionsText = localize('designer.actions', "More Actions");
+					const actionsColumn = new ButtonColumn({
+						id: 'actions',
+						iconCssClass: Codicon.ellipsis.classNames,
+						name: moreActionsText,
+						title: moreActionsText,
+						width: 100,
+						resizable: true,
+						isFontIcon: true
+					});
+					this._register(actionsColumn.onClick(async (e) => {
```

Comment:
is this async? There isn't an await below

Reply:
good catch, removed `async`